### PR TITLE
Add uptime-kuma

### PR DESCRIPTION
**Pull request by ketsapiwiq**
Latest CI says:
https://ci-apps-dev.yunohost.org/ci/job/3624

I normally fixed the linting things which should mean level 7 I hope! 🙏

### CODE REVIEW

File: apps.json [hunk 1]
```diff
@@ -3295,6 +3295,18 @@
         "state": "notworking",
         "url": "https://github.com/YunoHost-Apps/unbound_ynh"
     },
+    "uptime-kuma": {
+        "category": "system_tools",
+        "level": 7,
```
Comment (yalh76) [inline apps.json:3300]:
```suggestion
```

Level is not needed, it will be calculated automatically

File: apps.json [hunk 2]
```diff
@@ -3295,6 +3295,18 @@
...
+        "subtags": [
+            "monitoring"
+        ],
+        "branch": "master",
+        "revision": "1e274b81a2222208bd5bdf80372ae4d1c989ed98",
```
Comment (yalh76) [inline apps.json:3305]:
```suggestion
```
As master is the default branch and as it the default head that will be used, thoses parameters are not needed

File: apps.json [hunk 3]
```diff
@@ -3295,6 +3295,18 @@
...
+        "state": "working",
+        "maintained": true,
```
Comment (yalh76) [inline apps.json:3307]:
```suggestion
```
I's maintained by default

**Review by yalh76 — CHANGES_REQUESTED** [review]
Thanks for adding this package, just need few removal before merging :)

**Comment by ketsapiwiq** [comment]
Done! Thank you!

**Review by alexAubin — APPROVED** [review]

**Pull request merged**